### PR TITLE
Fix 'Shift+Backspace' to work the same as 'Backspace'

### LINKED ISSUE #896: Shift+Backspace should work the same as just Backspace
With the post-beta.3 PSReadLine, `Shift+Backspace` produces `^H`. This is a regression from beta.3, and `Shift+Backspace` should work just like `Backspace`.

### PR DESCRIPTION
**Pull request by daxian-dbw**
Fix #896
Make `Shift+Backspace` and `Shift+Escape` work the same as `Backsapce` and `Escape` on windows.

### CODE REVIEW

**Review by lzybkr — COMMENTED** [review]
Can you add a test?

File: test/KeyBindingsTest.cs [hunk 1]
```diff
@@ -47,5 +47,21 @@ public void ShowKeyBindings()
...
+            TestSetup(KeyMode.Cmd);
+            Test("aa", Keys("aaa", _.Shift_Backspace));
+        }
+
+        [SkippableFact]
+        public void ShiftEscape()
+        {
+            TestSetup(KeyMode.Cmd);
+
+            System.Threading.Thread.Sleep(10000);
```
Comment (lzybkr) [inline test/KeyBindingsTest.cs]: Remove or comment why this is necessary.

File: test/KeyInfo-en-US-windows.json [hunk 2]
```diff
@@ -1301,6 +1301,18 @@
         "ConsoleKey":  "UpArrow",
         "Modifiers":  "Shift"
     },
+    {
```
Comment (lzybkr) [inline test/KeyInfo-en-US-windows.json:1304]: The other keyboard layouts will need similar entries.

**Review by lzybkr — CHANGES_REQUESTED** [review]

File: test/KeyBindingsTest.cs (hunk 1 above)
Comment (daxian-dbw) [reply test/KeyBindingsTest.cs]: It was for diagnosis :) Already removed.

File: test/KeyInfo-en-US-windows.json (hunk 2 above)
Comment (daxian-dbw) [reply test/KeyInfo-en-US-windows.json:1304]: Updated. I installed the keyboard layouts and collect the data using `[Console]::ReadKey()`.

**Review by lzybkr — APPROVED** [review]

**Pull request merged**